src/index.js

```diff
@@ -259,56 +262,65 @@ class MiniCssExtractPlugin {
           const chunkMap = this.getCssChunkObject(chunk);
           if (Object.keys(chunkMap).length > 0) {
             const chunkMaps = chunk.getChunkMaps();
-            const linkHrefPath = mainTemplate.getAssetPath(
-              JSON.stringify(this.options.chunkFilename),
-              {
-                hash: `" + ${mainTemplate.renderCurrentHashCode(hash)} + "`,
-                hashWithLength: (length) =>
-                  `" + ${mainTemplate.renderCurrentHashCode(hash, length)} + "`,
-                chunk: {
-                  id: '" + chunkId + "',
-                  hash: `" + ${JSON.stringify(chunkMaps.hash)}[chunkId] + "`,
-                  hashWithLength(length) {
-                    const shortChunkHashMap = Object.create(null);
-                    for (const chunkId of Object.keys(chunkMaps.hash)) {
-                      if (typeof chunkMaps.hash[chunkId] === 'string') {
-                        shortChunkHashMap[chunkId] = chunkMaps.hash[
+            let chunkFilename;
+            try {
+              chunkFilename = JSON.stringify(
+                this.getFilename(chunk, this.options.chunkFilename)
+              );
+            } catch (err) {
+              throw new Error(
+                `Mini CSS Extract Plugin\n\nCouldn't stringify JSON for filename {Function}: ${err}\n`
+              );
+            }
+
+            const linkHrefPath = mainTemplate.getAssetPath(chunkFilename, {
+              hash: `" + ${mainTemplate.renderCurrentHashCode(hash)} + "`,
+              hashWithLength: (length) => `"
```

Comment:
```diff
- hashWithLength: (length) => {} 
+ hashWithLength (length)  {} 
```

Reply:
@michael-ciniawsky we need `eslint` rule in `webpack-defaults` for this :+1:

Reply:
Yep, `eslint-config-webpack` ;)